packages/utils/recursive-parser.ts

```diff
@@ -0,0 +1,51 @@
+import * as utils from "./ast-utils";
+import { INode } from "./NodePath";
+
+export default function recursiveTransform(j: any, ast: INode, key: string, value: any, action: string) {
```

Comment:
`boolean`, `string` etc type for value. You should create an interface for these types.